### ISSUE TITLE
Case data fixed fields grammar & spelling fixes

### ISSUE THREAD

**Issue by scottofletcher** [participedia/frontend#789]
Implement all changes listed in this document: https://docs.google.com/document/d/1D_LW96J3geVsTm0hJHpagSBY7fhKsUZFbjbdCEk1O6M/edit

**Comment by scottofletcher** [comment]
Also see #718 #436 #411

**Comment by scottofletcher** [comment]
@jesicarson @dethe @LLPrin Do you guys want me to make a new case field editor which reflects all these changes?

**Comment by jesicarson** [comment]
@scottofletcher do you mean a spreadsheet?

**Comment by scottofletcher** [comment]
@jesicarson Yes, or any other format that would be easier for you guys to use

**Comment by scottofletcher** [comment]
@jesicarson @dethe can I get confirmation on this? I'd be happy to make an up-to-date field editor spreadsheet like this one for methods (which is almost complete!!): https://docs.google.com/spreadsheets/d/1byKG6bewQu3NOS__Zqv2sgBNFoaLI5f5AuINzGMaI6k/edit#gid=678179084

**Comment by jesicarson** [comment]
I’ll have a look when I’m back online at the methods one. Have you flagged existing stuff that changes? 

I’m also wondering what the easiest way for Dethe/ Lydia to make all these edits would be or if it’s possible at all for me to get set up to do at least copy edits.. hmm. Let’s ask Dethe on the call tomorrow.

**Comment by scottofletcher** [comment]
@jesicarson no rush on the methods (there are still a couple things we have yet to sign off on). I didn't flag existing stuff that changes because we literally threw out every single field and started from scratch. There are also a lot of fields that are different depending on whether the user indicates they are entering a 'Tool/Technique' or a 'Method' (which Dethe said was ok. It is complicated though.)

Yes, I'm adding it to the D&T agenda right now! Will @LLPrin be on the call? Might be a good idea so Dethe doesn't have to take time out to relay everything
